homework_1/task_2/tax/testCases_test.go

```diff
@@ -0,0 +1,131 @@
+package tax_test
+
+import "code-cadets-2021/homework_1/task_1/tax"
+
+
+type calculateTaxTestCase struct {
+	inputValue float64
+	inputTaxBrackets []tax.TaxBracket
+
+	expectedOutput float64
+	expectingError bool
+}
+
+type getTaxBracketsTestCase struct {
+	inputFile string
+
+	expectedOutput []tax.TaxBracket
+	expectingError bool
+}
+
+
+func getCalculateTaxTestCases() []calculateTaxTestCase {
```

Comment:
All the test cases here cover the happy path only. Add some test cases where `inputTaxBrackets` are completely broken (e.g. thresholds are overlapping, tax brackets aren't properly sorted etc...).